### ISSUE TITLE
Weather plugin showing HTML tags instead of actual weather

### ISSUE THREAD

**Issue by Kristof-VD-Ouweland** [concerto/concerto-weather#44]
Related with issue 42 (which is currently closed)

Screens are showing HTML tags and codes for the weather plugin instead of the actual weather.
Weather plugin has been updated to 0.6 according to instructions in issue 42.

System info: 
- Ubuntu 16.04
- Installed via deb package
- Weather plugin updated via GIT 

![image](https://user-images.githubusercontent.com/28003415/31384340-da185e72-adbe-11e7-96a8-ff62aa293ea0.png)

Thanks for any advice.
Kristof

**Comment by mfrederickson** [comment]
@Kristof-VD-Ouweland did you reboot the server after updating the plugin?

**Comment by mfrederickson** [comment]
@Kristof-VD-Ouweland your screencapture above shows that the source for the weather plugin is rubygems and not git...
![image](https://user-images.githubusercontent.com/473165/31397548-c638721a-ad92-11e7-836c-56de81db3cad.png)

**Comment by Kristof-VD-Ouweland** [comment]
@mfrederickson 
- Yes the server was rebooted 
- I'll check but I did a manual update via git and set it back to rubygems.
I'll report in 10 minutes ;-)
Kristof

**Comment by Kristof-VD-Ouweland** [comment]
@mfrederickson 

To verify: 
- Plugin source has been set to GIT
- Reboot has been done of the server
- To be entirely sure, apache2 has been restarted

The other difference I see between our screenshots is that the plugin Gem version says (locked) or not.
Could that be something?


![image](https://user-images.githubusercontent.com/28003415/31445089-a615639c-ae9d-11e7-843e-a97bf7b3473a.png)

But still the HTML tags
![image](https://user-images.githubusercontent.com/28003415/31445108-b64d7c7c-ae9d-11e7-9824-d21de0ac542b.png)

**Comment by mfrederickson** [comment]
Could be a locked problem, I'm not sure.  Have you tried adding a new
weather content and see if it is fixed going forwards?

**Comment by Kristof-VD-Ouweland** [comment]
Yes and did a seperate restart of the apache2 daemon to be sure.

Grtn
Kristof VD Ouweland

**Comment by mfrederickson** [comment]
Can you search through your Gemfile.lock and send me any lines that mention
concerto-weather?

**Comment by Kristof-VD-Ouweland** [comment]
That would be following I assume:
`
root@kristof-test-001:/# cat /usr/share/concerto/Gemfile.lock | grep -i weather
 remote: https://github.com/concerto/concerto-weather.git
 concerto_weather (0.6)
 concerto_weather (= 0.6)!

`
